com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassRaytracingGBuffer.hlsl

```diff
@@ -27,23 +21,26 @@ void ClosestHitGBuffer(inout RayIntersectionGBuffer rayIntersectionGbuffer : SV_
     SurfaceData surfaceData;
     BuiltinData builtinData;
     bool isVisible;
-    GetSurfaceAndBuiltinData(fragInput, viewWS, posInput, surfaceData, builtinData, currentVertex, rayIntersectionGbuffer.cone, isVisible);
-
-    // Sometimes, we only  want to use the diffuse when we compute the indirect diffuse
-    if (_RayTracingDiffuseLightingOnly)
-    {
-        builtinData.bakeDiffuseLighting = float3(0.0, 0.0, 0.0);
-        builtinData.backBakeDiffuseLighting = float3(0.0, 0.0, 0.0);
-    }
-
+    RayCone cone;
+    cone.width = 0.0;
+    cone.spreadAngle = 0.0;
+    GetSurfaceAndBuiltinData(fragInput, -incidentDir, posInput, surfaceData, builtinData, currentVertex, cone, isVisible);
+    
+#ifdef MINIMAL_GBUFFER
+    // First we pack the data into the standard bsdf data
+    StandardBSDFData standardLitData;
+    ZERO_INITIALIZE(StandardBSDFData, standardLitData);
+    FitToMinimalLit(surfaceData, builtinData, fragInput.tangentToWorld[2], standardLitData);
+#else
     // First we pack the data into the standard bsdf data
     StandardBSDFData standardLitData;
     ZERO_INITIALIZE(StandardBSDFData, standardLitData);
     FitToStandardLit(surfaceData, builtinData, posInput.positionSS, standardLitData);
```

Comment:
To illustrate what I was mentioning previsouly, we could have

  FitToStandardLit(surfaceData, builtinData, posInput.positionSS, standardLitData);
#ifdef MINIMAL_GBUFFER
surfaceData.perceptualRoughness = 0.0;
    surfaceData.fresnel0 = 0.0;
    surfaceData.coatMask = 0.0;
(etc..)
#endif

thus no need to add a function in every material for minimal fit